libfwupdplugin/fu-kernel.c

```diff
@@ -305,3 +305,75 @@ fu_kernel_get_cmdline(GError **error)
 	return NULL;
 #endif
 }
+
+/**
+ * fu_kernel_set_cmdline:
+ * @arg: (not nullable): key to set
+ * @set_unset: set or unset the parameter
+ * @error: (nullable): optional return location for an error
+ *
+ * Set kernel parameters through grubby
+ *
+ * Returns: %TRUE if successful
+ *
+ * Since: 1.9.5
+ **/
+
+gboolean
+fu_kernel_set_commandline(const gchar *grubby_path,
+			  gboolean set_unset,
```

Comment:
I think two functions might be clearer, e.g. `fu_kernel_add_cmdline_arg()` and `fu_kernel_remove_cmdline_arg()`